docs/src/tut/docs.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: docs
+---
+
+# Get started
+
+The motivation behind this library is to provide a simple, easy way to provide generated date and time instances that are useful to your own domain.
+
+This library is relatively immature. Any issues, suggestions or criticisms are more than welcome.
```

Comment:
`This library is relatively immature` Not sure this statement is necessary, code looks to me as production ready as any other code. I think the fact that is a 0.1 indicates that it is still incubating. This sentences would make companies concerned about using it perhaps.